Rakefile

```diff
@@ -3,4 +3,10 @@
 require 'rubocop/rake_task'
 RuboCop::RakeTask.new
 
-task default: [:rubocop]
+begin
+  require 'rspec/core/rake_task'
+  RSpec::Core::RakeTask.new(:spec)
+rescue LoadError
+end
+
+task default: [:rubocop, :spec]
```

Comment:
This doesn't make sense to me.  Rubocop checks this repo.  But `spec` tests everything outside of this repo.

Reply:
not sure ?  spec runs all the tests I think... travis only does rubocop